tools/generator_python3.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python
+# coding=ascii
+###############################################################################
+# Copyright 2015-2016 University of Florida. All rights reserved.
+# This file is part of UF CTS-IT's NACCulator project.
+# Use of this source code is governed by the license found in the LICENSE file.
+###############################################################################
+
+import csv
+import os
+import re
+import sys
+# The output is too big for the command terminal, so I'm going to 
+# print everything to 'output.txt' located in the same directory.
+sys.stdout = open('output.txt', 'wt')
+
+class DynamicObject(object):
```

Comment:
Is this a conversion of `tools/generator.py` for python3? I guess it was missed somehow in the `2to3` conversion? If so, this needs to replace the original file.

Reply:
yes, it's a conversion. i will replace the original.